### ISSUE TITLE
Run eslint in project's subdir, resolved package report failure

### ISSUE THREAD

**Issue by huangyingjie** [benmosher/eslint-plugin-import#747]
If I run eslint in the project's root dir, everything goes right. However as the title describes, if i cd ./somedir, and  run eslint again, the resovled package cannot be found.

````js
  resolve: {
    alias: {
      helpers: path.resolve(__dirname, './client/helpers')
....
    },

````
This will throw error that says: cannot resolve `helpers/xxx`

**Comment by ljharb** [comment]
If `path.resolve` is what's issuing that error, that's coming from node itself, not from this plugin.
